src/components/ExpandedBankTransactionRow/ExpandedBankTransactionRow.tsx

```diff
@@ -241,18 +242,20 @@ export const ExpandedBankTransactionRow = forwardRef<SaveHandle, Props>(
       const endpoint = `/v1/businesses/${businessId}/bank-transactions/${bankTransaction.id}/metadata`
 
       if (showDescriptions && memoText != undefined) {
-        const headers = {
-          method: 'PUT',
-          headers: {
-            Authorization: `Bearer ${auth.access_token}`,
-            'Content-Type': 'application/json',
+        const result = await Layer.updateBankTransactionMetadata(
+          apiUrl,
+          auth.access_token,
+          {
+            params: {
+              businessId: businessId,
+              bankTransactionId: bankTransaction.id,
+            },
+            body: {
+              memo: memoText,
+            },
           },
-          body: JSON.stringify({
-            memo: memoText,
-          }),
-        }
-        const result = await fetch(apiUrl + endpoint, headers)
-        const resultJson = await result.json()
+        )
+        if (result.errors) console.error(result.errors)
```

Comment:
Do you intend to leave this in? Don't think this is our normal pattern unless I missed it elsewhere.